node/tests/swamp/config.go

```diff
@@ -9,19 +9,19 @@ import (
 	"github.com/celestiaorg/celestia-node/core"
 )
 
-// Config struct represents a set of pre-requisite attributes from the test scenario
-type Config struct {
+// InfraComponents struct represents a set of pre-requisite attributes from the test scenario
+type InfraComponents struct {
```

Comment:
You can omit Infra tbh. The full name would be then `swamp.Components` which is descriptive enough, but the idea to rename it good.